GenderPayGap.WebUI/Models/Report/ReportFiguresViewModel.cs

```diff
@@ -0,0 +1,139 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+using GenderPayGap.WebUI.Views.Models.ValidationAttributes;
+using GovUkDesignSystem;
+using GovUkDesignSystem.Attributes;
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace GenderPayGap.WebUI.Models.Report
+{
+    public class ReportFiguresViewModel : GovUkViewModel
```

Comment:
there are other validation rules to be added as well

Reply:
is the `ReturnViewModel` used anywhere?

Reply:
yup it's mainly in tests and in the viewing service